xarray/core/common.py

```diff
@@ -155,6 +155,9 @@ def __complex__(self: Any) -> complex:
     def __array__(self: Any, dtype: DTypeLike = None) -> np.ndarray:
         return np.asarray(self.values, dtype=dtype)
 
+    def get_duck_array(self):
+        return self.data
```

Comment:
Do we need this method in the public API for DataArray/Variable? How about just using `.data` instead for these cases?